### ISSUE TITLE
Add option to only build and publish to remote machine

### ISSUE THREAD

**Issue by vnbaaij** [radutomy/VSRemoteDebugger#7]
Sometimes you just want to deploy changes to the remote machine instead of starting a debug session. Code wise this is almost completely equal to the code that is already there. Just need to remove the `Debug()` call from the `Execute` method.

**Comment by vnbaaij** [comment]
Great, but....

In my opinion it would be better if this would be a separate menu option instead of doing this through the Settings. Doing it this way works but is less flexible. Suppose I'm working on a project and changing code and debugging. Then in a debug run I spot a typing error in a message or something small. If I then want to deploy just that small change without debugging, I first have to go through the Options dialog to do so. A separate option makes the flow a lot easier I think. Kind of like the F5 and CTRL+F5 options in VS itself.

**Comment by radutomy** [comment]
Agreed with you, but a separate menu option required a bit more coding effort than I had time for now. Noted for next release.

**Comment by Reveon** [comment]
Would it be possible to add this option to the project context menu and make it independent of Startup project? For example if there are more projects like server and client and you change something in the client then you have to switch startup project to client to deploy it and then switch again to server to test it. With context menu option you would just right click on the client project and hit deploy

**Comment by radutomy** [comment]
These are all good ideas, thank you for your feedback.
